tests/e2e/testing-manifests/tkg/tkg.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: run.tanzu.vmware.com/v1alpha1
+kind: TanzuKubernetesCluster
+metadata:
+  name: test-cluster-e2e-script-1
+  namespace: test-gc-e2e-demo-ns
+spec:
+  topology:
+    controlPlane:
+      count: 3
+      class: best-effort-xsmall # vmclass to be used for master(s)
```

Comment:
Sandeep had mentioned a different class that we can use, which will help with etcd crash issue

Reply:
For VMC I have done the testing with this vmclass and I'm planning to continue to use the same for VMC testing.